### DISCUSSION TITLE
tui-editor does not respect \r\n?

### DISCUSSION THREAD

**Discussion by vytautas-pranskunas-** [taiga-family/taiga-ui#9768] [Q&A] [answered]
I have tui-editor and patching value form value with \r\n\r\n inside. When adding this template html to simple textarea line breaks persist. However in tui-ediotr everything is in one line. How to make it respect \r\n\r\n?

**Comment by splincode** [comment]
Could you add stackblitz? What are you passing and what should be
https://stackblitz.com/github/taiga-family/editor-preview

**Reply by vytautas-pranskunas-** [reply]
it is loading but that it crashes. Cannot open but what i am trying to paste is for instance
test textr\n\r\ntest text 2

there is no empty line between two sentences

**Reply by splincode** [reply]
Maybe you can change `\n` to `<br/>` before?

https://taiga-family.github.io/editor/processing/cleanup-html

**Reply by splincode** [reply]
`Cannot open` strange, but I haven't problem

<img width="1727" alt="image" src="https://github.com/user-attachments/assets/8e8eb0ac-7bd9-45a9-b385-afe1200fc28e">

**Reply by vytautas-pranskunas-** [reply]
I cannot bevause this is how email templates are built and use across different platforms. But \r\n is standart markup. Dont you think it should be supported?

**Comment by splincode** [comment, marked as answer]
Move discussion to https://github.com/taiga-family/taiga-ui/issues/9780
